tests_e2e/tests/scripts/agent_persist_firewall-access_wireserver

```diff
@@ -31,22 +31,34 @@ function check_online
 {
     echo "Checking network connectivity..."
 
-    echo "Running ping to 8.8.8.8 option"
-    checks=0
-    while true; do
-        if ping 8.8.8.8 -c 1 -i .2 -t 30; then
-            echo "Network is accessible"
-            return 0
-        fi
-        checks=$((checks + 1))
-        if [ $checks -gt 10 ]; then
-            break
-        fi
+    echo "Connecting to ifconfig.io to check network connection"
+    if command -v curl >/dev/null 2>&1; then
+        curl --retry 5 --retry-delay 5 --connect-timeout 5 -4 ifconfig.io/ip
+    elif command -v wget >/dev/null 2>&1; then
+        wget --tries=5 --timeout=5 --wait=5 -4 ifconfig.io/ip
+    else
+        http_get.py "http://ifconfig.io/ip" --timeout 5 --delay 5 --tries 5
+    fi
 
+    if [[ $? -eq 0 ]]; then
+        echo "Network is accessible"
+        return 0
+    else:
```

Comment:
I think this `else:` may be a syntax error

Reply:
updated